fe/lib/components/LocationSuggest/components/MapSelect.tsx

```diff
@@ -0,0 +1,176 @@
+import { useQuery } from '@apollo/client';
+import {
+  Alert,
+  Box,
+  ButtonIcon,
+  Column,
+  Columns,
+  IconClear,
+  Stack,
+  Text,
+  useColor,
+  useResponsiveValue,
+} from 'braid-design-system';
+import { Map, Marker, Overlay, ZoomControl } from 'pigeon-maps';
+import React from 'react';
+import { useState } from 'react';
+import { InlineCode } from 'scoobie';
+import { useDebounce } from 'use-debounce';
+
+import { Location } from '../../../types/seekApi.graphql';
+import { formatPoint } from '../../../utils/formatPoint';
+
+import { NEAREST_LOCATIONS } from './queries';
+
+import * as styles from './styles.css';
+
+const DEFAULT_ZOOM = 10;
+
+interface Props {
+  debounceDelay?: number;
+  schemeId: string;
+  onLocationSelected: (location: Location) => void;
+  initialLocation: [number, number];
+}
+
+export const MapSelect = ({
```

Comment:
Yep I think that would be good. I think having a separate display of it the developer dashboard like you had built is good to draw more direct attention to the `nearestLocations` query. Although having it as a part of the autosuggest is good, it is secondary to the autosuggest functionality being shown as the main focus in this component in the dev dashboard

Reply:
I have made this into a standalone `LocationSelectMap` component that the autosuggest now imports and uses. I also added a story for it so it can be viewed in isolation in the storybook.